.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
```

Comment:
nit: you can add _assignees_ so that new PR will notify the team if you want.

Reply:
probably reviewers